browser/components/newtab/tests/xpcshell/test_RemoteNewTabLocation.js

```diff
@@ -0,0 +1,39 @@
+/* globals Assert, RemoteNewTabLocation, Services */
+"use strict";
+
+Components.utils.import("resource:///modules/RemoteNewTabLocation.jsm");
+Components.utils.import("resource://gre/modules/Services.jsm");
+Components.utils.importGlobalProperties(["URL"]);
+
+add_task(function* () {
+  let defaultHref = RemoteNewTabLocation.href;
+
+  Assert.ok(RemoteNewTabLocation.href, "Default location has an href");
+  Assert.ok(RemoteNewTabLocation.origin, "Default location has an origin");
+  Assert.ok(!RemoteNewTabLocation.overridden, "Default location is not overridden");
+
+  let testURL = new URL("https://example.com/");
+  let notificationPromise;
```

Comment:
because the variable is re-used further below.
instead of:
```javascript
let foo = bar;
...
...
...
foo = baz;
```
I like to do
```javascript
let foo;
foo = bar;
...
...
...
foo = baz;
```

This is to make it explicit that this variable will be re-used.